modular_skyrat/modules/sec_haul/code/peacekeeper/armadyne_clothing.dm

```diff
@@ -1,265 +0,0 @@
-/obj/item/clothing/under/rank/security/peacekeeper/armadyne
```

Comment:
I feel like the removal of everything Armadyne should go in a separate PR from the one that's changing all the guns. There's so much going on here that it's incredibly difficult to give anything the proper time to review that it should have. I'd like if you could atomize those non-gun-related changes out for a different PR.

Reply:
Ok